### DISCUSSION TITLE
Trouble deploying Nestjs app on Vercel (Zeit) with Git Integration

### DISCUSSION THREAD

**Discussion by jacobdo2** [vercel/vercel#4438] [Help] [answered]
I have a github action set up to deploy a Nestjs app on Vercel, but it is giving me the following error:
`You defined 1 build that did not match any source files (please ensure they are NOT defined in .vercelignore):`

and there is nothing below the message.

My `now.json` file looks as follows:

```json
{
  "version": 2,
  "builds": [
    {
      "src": "./dist/main.js",
      "use": "@now/node"
    }
  ],
  "routes": [
    {
      "src": "/(.*)",
      "dest": "./dist/main.js"
    }
  ]
}
```

I am able to deploy fine from command line, but I would prefer to do it through github integration. The build does not come up under deployments in Vercel, so I am not sure as to how I can debug this myself.

**Comment by paulogdm** [comment, marked as answer]
Vercel is a static-first platform and we don't recommend that you deploy Nest.js or backend frameworks to our platform. Please consider using another platform for this type of deployment.

This file seems like a generated one and it probably doesn't exist in your repository:
```
  "builds": [
    {
      "src": "./dist/main.js",
      "use": "@now/node"
    }
  ],
```
